app/src/main/res/values/strings.xml

```diff
@@ -826,7 +821,14 @@
 
     <string name="multimeter_dialog_heading">Multimeter</string>
     <string name="multimeter_dialog_text">Use PV1, PV2, PV3 Connect the SEN and CAP pins to the resistor and capacitor respectively as shown in the figure to measure the resistance or capacitance of the connected element.</string>
-    <string name="multimeter_dialog_description">\u2022 To measure the voltage directly connect the power source(eg PV1,PV2 and PV3) to the channel pins as shown in the figure.\n\n\u2022 Similarly to measure frequency or count pulse connect SQ1,SQ2,SQ3 or SQ4 pins to ID1,ID2,ID3 or ID4.</string>
+    <string name="multimeter_dialog_description">\u2022 To measure the voltage directly connect the power source(eg PV1,PV2 and PV3) to the channel pins as shown in the figure.\n\n\u2022 Similarly to measure frequency or count pulse connect SQ1,SQ2,SQ3 or SQ4 pins to ID1,ID2,ID3 or ID4.\n\n
+        <b>Different Sections</b>\n\n
+        <b>Resistance:</b> Measures the resistance of the element connected between the SEN and GND pin of the PSLab device.\n\n
+        <b>Capacitance:</b> Measures the capacitance of the element connected to the CAP pin of the PSLab device.\n\n
+        <b>Voltage:</b> Measures the voltage of any elements connect across any of the channel pins and other pins such as AN8.\n\n
```

Comment:
Similarly in this and the rest. We have to connect one end to measuring pin and the other end to GND